README.md

```diff
@@ -151,6 +72,31 @@ Open the project's folder and run one of the commands.
 
 Administrator account is _admin_:_admin_.
 
+### How to build Open Y project ?
```

Comment:
Let's update the whole documentation page to make it simple according to our discussion.

Reply:
Fixed